ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -288,7 +293,7 @@ StackView {
                 // Minted tokens will use ALWAYS the same chain where the owner token was deployed.
                 chainId: newTokenPage.ownerTokenChainId
                 chainName: newTokenPage.chainName
-                chainIcon: newTokenPage.chainIcon
+                chainIcon: newTokenPage.chainIconownerTokenChainId
```

Comment:
Typo???
```suggestion
                chainIcon: newTokenPage.chainIconOwnerTokenChainId
```

Reply:
That's an oopise, I guess I pasted here by accident. Reverting